src/cephadm/cephadm

```diff
@@ -5197,6 +5197,14 @@ def extract_uid_gid_monitoring(ctx, daemon_type):
     return uid, gid
 
 
+def apply_extra_container_args(ctx, c):
+    # type: (CephadmContext, CephContainer) -> CephContainer
+    if 'extra_container_args' in ctx and ctx.extra_container_args:
+        logger.info(ctx.extra_container_args)
```

Comment:
Shouldn't this be something like `logger.info("found extra cotainer args: %r", ctx.extra_container_args)`

Reply:
This was actually a log statement for testing I forgot to remove. This stuff should already be making its way into the debug logs as part of the _run_cephadm command logging. Thanks for catching.